dds_cli/options.py

```diff
@@ -187,6 +187,35 @@ def token_path_option(
     )
 
 
+def unit_prefix_option(
+    long="--unit-prefixes",
+    short="-up",
+    name="unitprefix",
+    choices=["auto", "auto-iec", "const", "const-iec", "off"],
+    case_sensitive=False,
+    default="auto",
+    required=False,
+    help_message="Use unit prefixes in order to reduce the number of digits. For Bytes, the SI prefixes can be replaced by powers of 1024 (IEC prefixes).",
```

Comment:
```suggestion
    help_message=("Unit formatting in the Size column. "auto": Use a consistent unit for all projects unless the variation is too big, in SI units (base 10). "auto-iec": Use consistent unit for all projects unless the variation is too big, in IEC units (base 2). "const": Use consistent unit for all projects, in SI units (base 10). "const-iec": Use consistent unit for all projects, in IEC units (base 2). "off": Display the project sizes in bytes, no further unit simplification.
```

Reply:
So are you fine with such a comprehensive help message? I was thinking those should be more brief?

Reply:
Yeah. Tried now and we should keep it brief. Will change my suggestion.

Reply:
Done.

Reply:
I am not strictly opposed to changing the text, but I find this suggestion unfortunately not helpful:

- It is not the unit formatting, but the use of unit metric prefixes that is configured.
- It is not exclusively the `Size` column that is affected, but `Size`, `Usage` and `Cost `.
- _See documentation for more information_ is relatively self-evident. I would presume that those help texts are in general read mostly by people who per se know how the system works, but the meaning of one particular config option has escaped them at the moment. 

What do you think of:

 _Configure the output's metric unit prefixes: Consistent scaling is enforced with const, auto optimizes for fewer digits. The -iec modifier allows to use binary instead of decadic prefixes for bytes_.
 
 ?

Reply:
Ok fair enough.
I don't think most people understand what "Configure the output's metric unit prefixes" means though, for example. So I think we need a more user friendly text.

Reply:
Ok, how about: "_Adjust units to improve readability, either consistently with 'const' or adaptively with 'auto'. Prefixes are either decadic (Kilo-,Mega-,Giga-) by default or binary (Kibi-,Mebi-,Gibi-) when specifying '-iec'._"

Reply:
My try:
__Configure_ the output's unit prefixes: const results in identical unit prefixes between rows, auto optimizes for fewer digits. The -iec modifier allows to use binary instead of decadic prefixes for bytes._

Please correct me if this is in fact incorrect though!

Reply:
> My try: __Configure_ the output's unit prefixes: const results in identical unit prefixes between rows, auto optimizes for fewer digits. The -iec modifier allows to use binary instead of decadic prefixes for bytes._
> 
> Please correct me if this is in fact incorrect though!

Yes, I like this one. Lets use this 👍🏻  @MatthiasZepper I'm assuming the thumbs up from your side is that it's correct so you can ignore my suggestion and just use when Johannes said here, ok?

Reply:
Yes, the thumbs up referred to the correctness. However, does this suggestion sufficiently address your concern that people do not know what "_Configure the output's metric unit prefixes"_ means? Except for the word "_metric_" it is pretty much similar or am I missing something. 

I tried to include examples (Kilo,Mega,Giga) to exemplify in case someone doesn't know what a unit prefix is and stepped back from detail description of the options to just call the formatting _adaptive_ and _consistent_. Does any of those sound like an additional improvement or should be exactly go with Johannes proposal?

Reply:
@MatthiasZepper No it doesn't. But I mean that this is clearer at least. Having those examples that you mention would make it better.

So in that case
```suggestion
    help_message="Configure the output's unit prefixes (Kilo, Mega, Giga): const results in identical unit prefixes between rows, auto optimizes for fewer digits. The -iec modifier allows to use binary instead of decadic prefixes for bytes.",
```
?

Reply:
I have slightly rephrased it, but in general it would work. The length of both proposals with 222 respectively 225 characters might be an issue, though...